README.md

```diff
@@ -1,5 +1,84 @@
-# beman.inplace\_vector: A dynamically-resizable vector with fixed capacity and embedded storage
-
 <!--
 SPDX-License-Identifier: <SPDX License Expression>
 -->
+
+# beman.inplace\_vector
+
+A dynamically-resizable vector with fixed capacity and embedded storage
+
+This repository implements `std::inplace_vector` extension targeting C++26.
+The `beman.inplace_vector` library aims to evaluate the stability, 
+the usability, and the performance of these proposed changes before they 
+are officially adopted by WG21 into the C++ Working Draft. 
+Additionally, it allows developers to use these new features before they 
+are implemented in major standard library compilers.
```

Comment:
```suggestion
A dynamically-resizable vector with fixed capacity and embedded storage
```

I think we can skip the boilerplate since it is true of all beman libraries.

Reply:
Reasonable.